compose/config.go

```diff
@@ -99,6 +99,18 @@ type Config struct {
 	// UseLegacyErrorFormat controls whether the legacy error format (with `error_debug`, `error_hint`, ...)
 	// should be used or not.
 	UseLegacyErrorFormat bool
+
+	// JWTSkipClientAuth indicates, if client authentication can be skipped, when using jwt as assertion.
+	JWTSkipClientAuth bool
+
+	// JWTIDOptional indicates, if jti (JWT ID) claim required or not in JWT.
+	JWTIDOptional bool
```

Comment:
Given that we have many forms of JWT, I would prefix these fields with `GrantTypeJWTBearer`

Reply:
ok